server/Sample/sfreerdp.c

```diff
@@ -918,6 +919,7 @@ int main(int argc, char* argv[])
 	}
 
 	WTSRegisterWtsApiFunctionTable(FreeRDP_InitWtsApi());
+	winpr_InitializeSSL(WINPR_SSL_INIT_DEFAULT);
```

Comment:
This seems analogous to how it is being done in the shadow server currently. Should this sample server as well as the shadow server have the SSL initialization pushed down into freerdp_listener_new() perhaps?